notify.go

```diff
@@ -0,0 +1,39 @@
+package autonat
+
+import (
+	"time"
+
+	inet "github.com/libp2p/go-libp2p-net"
+	ma "github.com/multiformats/go-multiaddr"
+)
+
+var _ inet.Notifiee = (*AmbientAutoNAT)(nil)
+
+func (as *AmbientAutoNAT) Listen(net inet.Network, a ma.Multiaddr)      {}
+func (as *AmbientAutoNAT) ListenClose(net inet.Network, a ma.Multiaddr) {}
+func (as *AmbientAutoNAT) OpenedStream(net inet.Network, s inet.Stream) {}
+func (as *AmbientAutoNAT) ClosedStream(net inet.Network, s inet.Stream) {}
+
+func (as *AmbientAutoNAT) Connected(net inet.Network, c inet.Conn) {
+	p := c.RemotePeer()
+
+	go func() {
+		// add some delay for identify
+		time.Sleep(500 * time.Millisecond)
```

Comment:
Make configurable

Reply:
done in https://github.com/libp2p/go-libp2p-autonat/pull/1/commits/6a3a9cbab7396dbca42e9ef214435e3a6fab3af6